File: fibonacci.c
#include <stdio.h>
#include "stdlib.h"
#include "string.h"
int fibonacci(int x);
int main(void) {
  int i;
  for(i = 0; i<11; i++) {
    printf("%d\n",fibonacci(i));
  }
  return 0;
}

int fibonacci(int x) {
  if(x == 0) {
    return 0;
  }
  if(x == 1) {
    return 1;
  }
  return fibonacci(x-1) + fibonacci(x-2);
}

File: fibonacci.js
function fibonacci(x) {
	if(x == 0) {
		return 0;
	}
	if(x == 1) {
		return 1;
	}
	return (fibonacci(x-1) +fibonacci(x-2));
}

function getFibonacciSeries() {
	//get fibonacci series for numbers between 0 and 10
	for(var i = 0; i< 11; i++) {
		console.log("%d", fibonacci(i));
	}
}